common/v2/features/DeFiZap/components/ProtocolTagsList.tsx

```diff
@@ -8,7 +8,7 @@ interface Props {
 
 const ProtocolTagsList = ({ platformsUsed }: Props) => {
   const platformTags = platformsUsed.map(platform => ({
-    tagText: platform,
+    tagText: platform.toUpperCase(),
     color: getPlatformColor(platform)
```

Comment:
Compound color is off - it should be a light green (`#B3DD87`) according to the design.